src/components/form/row.vue

```diff
@@ -146,7 +162,10 @@ export default {
     "lastSubmission": "(last {dateTime})",
     "action": {
       "fill": "Fill Form"
-    }
+    },
+    "formClosedTip": "This Form is closed. It is no longer accepting submissions.",
+    "formClosingTip": "This Form is closing. It is not visible on new devices, but it is still accepting submissions.",
```

Comment:
We often capitalize Central-specific terms, like "Submissions", "Closed", and "Closing". I'm also wondering whether we should try to match the language in `FormOverviewRightNow` (that might be an question for @issa-tseng):

https://github.com/getodk/central-frontend/blob/fb714d1a7643815a78af1fb8d662bceec668710f/src/components/form/overview/right-now.vue#L118-L120